src/f5_tts/infer/infer_cli.py

```diff
@@ -207,16 +207,28 @@ def main_process(ref_audio, ref_text, text_gen, model_obj, mel_spec_type, remove
     if generated_audio_segments:
         final_wave = np.concatenate(generated_audio_segments)
 
+        # Save final concatenated audio
         if not os.path.exists(output_dir):
             os.makedirs(output_dir)
 
+        # Save entire wave file
         with open(wave_path, "wb") as f:
             sf.write(f.name, final_wave, final_sample_rate)
             # Remove silence
             if remove_silence:
                 remove_silence_for_generated_wav(f.name)
             print(f.name)
 
+        # Ensure the chunk directory exists
+        chunk_dir = "/content/aud/chunk"
```

Comment:
os.path.join(output_dir,chunk_dir) will be a good choice

Reply:
actully im in running colab so ... its aceddently push

Reply:
if you don't mind, I could add this pr in https://github.com/SWivid/F5-TTS/pull/518

Reply:
its work like save the chucks and if if len(audio) / final_sample_rate < 0.8 * ref_audio_length: #the output lenth is not match 80% of ref_audio is reganarate

Reply:
its more we can improve it like calculating the ref audios paramets and config the gen_audio like that